terms.html

```diff
@@ -169,8 +169,12 @@
 <a>verifiable presentation</a> for processing. Other specifications might refer
 to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
   </dd>
-  <dt><dfn data-lt="URI|URIs">URI</dfn></dt>
+  <dt><dfn data-lt="URL|URLs">URL</dfn></dt>
   <dd>
-A Uniform Resource Identifier, as defined by [[RFC3986]].
+A Uniform Resource Locator, as defined by [[URL]]. This specification does
+not use the term URI or IRI because those terms have been deemed to be confusing
+to Web developers. This specification uses the phrase "nondereferenceable URL"
+when referring to the concept of a URI or IRI, as defined in [[?RFC3986]] and
+[[?RFC3987]], respectively.
```

Comment:
Why are these references non-normative?
[[?RFC3986]] and [[?RFC3987]]

Reply:
The WHATWG URL specification supersedes the URI and IRI specifications and therefore we no longer have any normative dependence on the IETF URI and IRI specifications. Yes, the horror :), but W3C and IETF lost that fight years ago and now WHATWG is the maintainer of "how to process a URI/IRI/URL".

We now depend on the WHATWG URL specification for all normative language on how to treat and process URLs (and IRIs and URIs). The WHATWG URL specification is what's implemented by the vast majority of modern URI/IRI/URL processors/resolvers, AFAIK.